tests/fx/converters/test_activations_fx.py

```diff
@@ -3,6 +3,42 @@
 from fx_test_utils import randbounds, FuncModule, MethodModule, convert_to_mgx, verify_outputs
 
 
+# TODO: test with 3 or more dimensions
+@pytest.mark.parametrize('inp_size, weight_size', [((3, 5), 5), ((3, 5), 0)])
+def test_nll_loss_forward_fx(inp_size, weight_size):
+    # weight_size should be either inp_size[1], aka C or number of classes
+    # or else 0.
+    # if weight_size = 0 , then pass weight=None, module should default weights to 1
+
+    # C is the number of classes and weights
+    C = inp_size[1]
+    if len(inp_size) == 2:
+        target_size =  [inp_size[0]]
+    else:  # k-dimensional inputs
+        #   remove C at index 1, then the rest is the shape of target
+        target_size = inp_size[:1] + inp_size[2:]
+    target = torch.randint(C, target_size).cuda()
+
+    # TODO: get correct weight size for k-dimensional case
+    weight = torch.rand(weight_size, dtype=torch.float).cuda()
+    if weight_size == 0:
+        weight = None
+
+
+    inp = torch.randn(inp_size, dtype=torch.float).cuda()
+
+    mod1 = FuncModule(torch.nn.functional.nll_loss, target=target, weight=weight,
+                       reduction = 'mean', ignore_index = -100)
+    mod2 = FuncModule(torch.nn.functional.nll_loss, target=target, weight=weight,
+                       reduction = 'sum', ignore_index = -100)
+    mod3 = FuncModule(torch.nn.functional.nll_loss, target=target, weight=weight,
+                       reduction = 'none', ignore_index = -100)
+
+    for mod in [mod1, mod2, mod3]:
+        mgx_mod = convert_to_mgx(mod, [inp])
+        verify_outputs(mod, mgx_mod, [inp])
```

Comment:
```suggestion
    mod = FuncModule(torch.nn.functional.nll_loss, target=target, weight=weight,
                       reduction=reduction, ignore_index = -100)

    mgx_mod = convert_to_mgx(mod, [inp])
    verify_outputs(mod, mgx_mod, [inp])
```

The reduction parameter here should be parametrized. Right now even if one of the reduction type fails, the whole test will fail and you will have no idea which one of these modules failed.